localisation/wc_religions.csv

```diff
@@ -147,7 +147,7 @@ INSANITY;Insanity;D
 THE_OLD_GODS;the Old Gods;les Dieux tr�s anciens;die alten G�tter;;;;;;;;;;;x
 voidfarer;Voidfarer;vide astral;Astrale;;;;;;;;;;;x
 forsaken_cult;Forgotten Shadow;ombre-oubli�e;Vergessener Schatten;;;;;;;;;;;x
-voidfarer_DESC;While dangerous, void magic can be carefully used for the benefit of its users and their allies. Void practictioners have devoted their lives to the study and use of the void for good.;La magie du Vide est tr�s dangereuse, mais elle peut �tre utilis�e en faveur de soi-m�me et des autres, si l'on est suffisamment prudent. Il ne sert � rien d'abandonner un outil aussi utile et de poursuivre les pr�tres de l'ombre. Les partisans de cette religion ont consacr� leur vie � l'�tude et � l'utilisation du Vide pour le Bien.;Obwohl sie gef�hrlich ist, kann Leeren-Magie vorsichtig zum Vorteil ihrer Verwender und deren Verb�ndeten eingesetzt werden. Die Anh�nger dieser Religion haben ihr Leben dem Studium und der Nutzung der Leere f�r das Gute gewidmet.;;;;;;;;;;;x
+voidfarer_DESC;While dangerous, void magic can be carefully used for the benefit of its users and their allies. Some of the Void practictioners have devoted their lives to the study and use of the void for good, such as the denizens of the Telogrus Rift, although there are some that are much more morally ambiguous than their peers, such as the K'areshi ethereals. Nevertheless, they can all enjoy the benefits of the Void without interference from the Old Gods, but still risk being corrupted by the malevolent whispers.;La magie du Vide est tr�s dangereuse, mais elle peut �tre utilis�e en faveur de soi-m�me et des autres, si l'on est suffisamment prudent. Il ne sert � rien d'abandonner un outil aussi utile et de poursuivre les pr�tres de l'ombre. Les partisans de cette religion ont consacr� leur vie � l'�tude et � l'utilisation du Vide pour le Bien.;Obwohl sie gef�hrlich ist, kann Leeren-Magie vorsichtig zum Vorteil ihrer Verwender und deren Verb�ndeten eingesetzt werden. Die Anh�nger dieser Religion haben ihr Leben dem Studium und der Nutzung der Leere f�r das Gute gewidmet.;;;;;;;;;;;x
```

Comment:
Changed the start of the second sentence a little to make it less clunky, corrected the spelling of practitioners. Also, is it proper to refer to the Telogrus Rift, given that they don't appear in GoA?
```suggestion
voidfarer_DESC;While dangerous, void magic can be carefully used for the benefit of its users and their allies. Some Void practitioners have devoted their lives to the study and use of the void for good, such as the denizens of the Telogrus Rift, although there are some that are much more morally ambiguous than their peers, such as the K'areshi ethereals. Nevertheless, they can all enjoy the benefits of the Void without interference from the Old Gods, but still risk being corrupted by the malevolent whispers.;La magie du Vide est tr�s dangereuse, mais elle peut �tre utilis�e en faveur de soi-m�me et des autres, si l'on est suffisamment prudent. Il ne sert � rien d'abandonner un outil aussi utile et de poursuivre les pr�tres de l'ombre. Les partisans de cette religion ont consacr� leur vie � l'�tude et � l'utilisation du Vide pour le Bien.;Obwohl sie gef�hrlich ist, kann Leeren-Magie vorsichtig zum Vorteil ihrer Verwender und deren Verb�ndeten eingesetzt werden. Die Anh�nger dieser Religion haben ihr Leben dem Studium und der Nutzung der Leere f�r das Gute gewidmet.;;;;;;;;;;;x
```